torch/nested/__init__.py

```diff
@@ -187,3 +188,69 @@ def nested_tensor(tensor_list, *, dtype=None, layout=None, device=None, requires
         return nt
     else:
         raise RuntimeError(f"Specified layout is unsupported for nested tensors: {layout}")
+
+
+def narrow(tensor: Tensor, dim: int, start: Union[int, Tensor], length: Union[int, Tensor], layout=torch.strided) -> Tensor:
```

Comment:
Hm, did the documentation for this make it onto the website?

Reply:
I might have forgotten to commit the changes to the docs adding this function, as I don't see it in the commit